WalletWasabi.Gui/Behaviors/MnemonicSuggestionBehavior.cs

```diff
@@ -2,20 +2,22 @@
 using Avalonia.Controls;
 using Avalonia.Xaml.Interactivity;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
+using WalletWasabi.Gui.Tabs.WalletManager;
```

Comment:
Iv never used it but Avalonia has an inbuilt auto completing textbox control, this might perhaps be more suitable?

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/AutoCompleteBox.cs

It is also in the control catalog if you want to evaluate the way it works.